lib/ain-evm/src/evm.rs

```diff
@@ -326,4 +353,33 @@ impl EVMServices {
 
         Ok(())
     }
+
+    pub fn counter_contract() -> Result<CounterContractInfo, Box<dyn Error>> {
+        let address = *CONTRACT_ADDRESSES.get(&Contracts::CounterContract).unwrap();
+        let bytecode = ain_contracts::get_counter_bytecode()?;
+        let (_, latest_block_number) = SERVICES
+            .evm
+            .block
+            .get_latest_block_hash_and_number()
+            .unwrap_or_default();
+        let count = U256::from(
+            SERVICES
+                .evm
+                .core
+                .get_storage_at(address, U256::one(), latest_block_number)?
+                .unwrap_or_default()
```

Comment:
Should we be propagating this error?

Reply:
This is not an error. In the initial state where the storage is unset for the address, we want to get the default value instead of panicking.